core/cloudflow-localrunner/src/main/scala/cloudflow/localrunner/LocalRunner.scala

```diff
@@ -80,13 +82,16 @@ object LocalRunner extends StreamletLoader {
     JRuntime.getRuntime.addShutdownHook(shutdownHook(fos))
 
     Console.withOut(fos) {
-      System.setOut(new PrintStream(fos))
-      readDescriptorFile(appDescriptorFilename) match {
-        case Success(applicationDescriptor) ⇒
-          run(applicationDescriptor, localConfig)
-        case Failure(ex) ⇒
-          log.error(s"Failed JSON unmarshalling of application descriptor file [${appDescriptorFilename}].", ex)
-          System.exit(1)
+      Console.withErr(fos) {
+        System.setOut(new PrintStream(fos))
+        System.setErr(new PrintStream(fos))
+        readDescriptorFile(appDescriptorFilename) match {
+          case Success(applicationDescriptor) ⇒
+            run(applicationDescriptor, localConfig)
+          case Failure(ex) ⇒
+            log.error(s"Failed JSON unmarshalling of application descriptor file [${appDescriptorFilename}].", ex)
+            System.exit(1)
```

Comment:
Would it be enough to throw the exception instead of abruptly exiting?
(this is not introduced in this PR, so no strict need to change it :-) )

Reply:
The LocalRunner is actually spawned as a separate JVM, so exit is probably best.